freqtrade/rpc/telegram.py

```diff
@@ -618,6 +621,12 @@ def _balance(self, update: Update, context: CallbackContext) -> None:
                 else:
                     output += curr_output
 
+            if total_dust_balance > 0:
+                output += (
+                    f"*{total_dust_currencies} Other Currencies:*\n"
```

Comment:
Perhaps mention that these currencies are below the `balance_dust_level` somehow via the output?

Reply:
*72 Other currencies (<1 USDT):
Est. USDT: 133.967*

like this, ok?

Reply:
looks good!